### DISCUSSION TITLE
OOP2

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2177] [Show and tell]
To update OOP2, follow the instructions here:

https://xdrip.readthedocs.io/en/latest/use/OOP/#oop2

**Comment by Navid200** [comment]
@psonnera

**Reply by psonnera** [reply]
@Navid200  Do you think I should specify that you need to uninstall the previous version and install the new one?

**Reply by Navid200** [reply]
That's a good idea.
It's my experience that if there is a way for things to go wrong, a user will end up exactly doing just that!

**Reply by tzachi-dar** [reply]
There is no need to uninstall the previous one. Just install the new one over it.

**Reply by psonnera** [reply]
Great. Will write that. Thanks
